<launch>
	<arg name="robot" default="armrobot"/>
	
	
	<node pkg="grasp_template_planning" type="howto_extract_templates" name="howto_extract_templates" output="screen">
		<rosparam file="$(find feature_learning)/config/template_config_$(arg robot).yaml" command="load"/>
	</node>
</launch>